tests/test_helpers.py

```diff
@@ -21,9 +21,10 @@
       "-a","Kurt Godel", "MC Escher", "Johann Sebastian Bach","-c", "Bitcoin",
       "--other_agencies", "Flatland","-n", "this is a sample added proposal"],
      "20_anewproposal has been added in proposals\n"),
-    (["helper", "a_expense", "159.18", "google", "Payment to Google Cloud to pay for cloud services",
-      "-b", "g", "dmref15", "mrsec14", "-d", "2020-06-18"],
-     "Payment to google has been added to expenses\n"),
+    (["helper", "a_expense", "159.18", "google", "Payment to google cloud",
```

Comment:
let's make it a bit more realistic, since these exemplars also form part of the documentation for users.  Since this is a travel expense, something like "Invited talk in timbuktoo" would be better.

Reply:
done